lib/routes/cjlu/yjsy/index.ts

```diff
@@ -80,22 +128,31 @@ async function handler(ctx) {
 
     const items = await Promise.all(
         list.map((item) =>
-            cache.tryGet(item.link, async () => {
-                if (!item.link || item.link === host) {
-                    return item;
-                }
+            cache.tryGet(
+                item.link,
+                async () => {
+                    if (!item.link || item.link === host) {
+                        return item;
+                    }
 
-                const res = await ofetch(item.link, {
-                    responseType: 'text',
-                });
-                const $ = load(res);
+                    const res = await ofetch(item.link, {
+                        responseType: 'text',
+                        headers: {
+                            ...headers,
+                            Cookie: cookieString,
+                            Referer: url,
+                        },
+                    });
+                    const $ = load(res);
 
-                const content = $('#vsb_content').html() ?? '';
-                const attachments = $('form[name="_newscontent_fromname"] div ul').html() ?? '';
+                    const content = $('#vsb_content').html() ?? '';
+                    const attachments = $('form[name="_newscontent_fromname"] div ul').html() ?? '';
 
-                item.description = `${content}<br>${attachments}`;
-                return item;
-            })
+                    item.description = `${content}<br>${attachments}`;
+                    return item;
+                },
+                24 * 60 * 60 // cache post content for 1 day
```

Comment:
Do not hard-code the cache duration. The value is expected to be changed through [environment variable](https://github.com/DIYgod/RSSHub/blob/ddf62a3269b798b91380b73573e3a6cbc7be6ad2/lib/config.ts#L501).